client/src/components/Profile.jsx

```diff
@@ -0,0 +1,150 @@
+        this.setState({Edit: true});
+    }
+
+  render() {
+      let avatar_image;
+      if (this.state.user.avatar == null) {
+          avatar_image = <img src="http://dialpharma.com/media/img/default_profile.png" alt="Default icon" />
+      }
+      else {
+          avatar_image = <img src={this.state.user.avatar} alt="Avatar icon" />
+      }
+      let page
+      if (this.state.Edit === false) {
+          page = <div id="main">
+            <Row>
+                <Col s={6}>
+                    <Row>
+                          {avatar_image}
+                    </Row>
+                </Col>
+                <Col s={2}>
+                    <Row s={1}><h4>Nickname:</h4></Row>
+                    <Row s={1}><h4>Email:</h4></Row>
+                    <Row s={1}><h4>First Name: </h4></Row>
+                    <Row s={1}><h4>Last Name: </h4></Row>
+                    <Row s={1}><h4>Location: </h4></Row>
+                </Col>
+                <Col s={3}>
+                    <Row s={1}><h4>{this.state.user.nickname}</h4></Row>
+                    <Row s={1}><h4>{this.state.user.email}</h4></Row>
+                    <Row s={1}><h4>{this.state.user.first_name}</h4></Row>
+                    <Row s={1}><h4>{this.state.user.last_name}</h4></Row>
+                    <Row s={1}><h4>{this.state.user.location}</h4></Row>
+                    <Row s={1}><Button waves='light' onClick={this.handleClick_Edit}>Edit</Button></Row>
+                </Col>
+            </Row>
+            <br />
+          </div>;
+      }
+      else if (this.state.Edit === true) {
+          page = <Row>
+              <Input s={12} id="email" label="Email" placeholder={this.state.user.email} onChange={this.onChangeHandler} />
+              <Input s={6} id="f_name" label="First Name" placeholder={this.state.user.first_name} onChange={this.onChangeHandler} />
+              <Input s={6} id="l_name" label="Last Name" placeholder={this.state.user.last_name} onChange={this.onChangeHandler} />
+              <Input s={12} id="nickname" label="Nickname" placeholder={this.state.user.nickname} onChange={this.onChangeHandler} />
+              <Input s={12} id="location" label="Location" placeholder={this.state.user.location} onChange={this.onChangeHandler} />
+              <Input s={6} id="new_password" label="New password" type="password" onChange={this.onChangeHandler} />
+              <Input s={6} id="repeat_password" label="Repeat password" type="password" onChange={this.onChangeHandler} />
+              <Input id="image" type='file' />
+              <br />
+              <Button waves='light' onClick={this.UpdateClick}>Save</Button>
+              <Button waves='light' onClick={this.handleClick_Main}>Back</Button>
+        </Row>;
+      }
+    return (
+      <div>
+        <div>
+          <h1>Profile</h1>
+        </div>
+        <ColoredLine color="teal" />
+        {page}
+      </div>
+    );
+  }
+}
+
+const ColoredLine = ({ color }) => (
```

Comment:
I don't think you need to define a new component. Also NEVER write the css properties here. Use .css file for this